media/inf/shortcuts.inf

```diff
@@ -1296,58 +1296,58 @@ README_TITLE=De citit
 README_DESC=Fișier de informare ReactOS
 
 CMD_TITLE=Linie de comandă
-CMD_DESC=Mediu textual cu operații lansate de la linia de comandă.
+CMD_DESC=Mediu textual cu operații lansate de la linia de comandă
 EXPLORER_TITLE=Explorator ReactOS
-EXPLORER_DESC=Program de explorare a fișierelor în ReactOS.
+EXPLORER_DESC=Program de explorare a fișierelor în ReactOS
```

Comment:
In some other languages there are excessive dots at the end as well, I think those should be removed once and everywhere.

Reply:
I understand, i'll do it